TestFoundation/TestNSString.swift

```diff
@@ -401,7 +401,7 @@ class TestNSString : XCTestCase {
         var buf : [UInt8] = []
         buf.reserveCapacity(reservedLength)
         var usedLen : CFIndex = 0
-        buf.withUnsafeMutableBufferPointer { p in
+        _ = buf.withUnsafeMutableBufferPointer { p in
```

Comment:
I think the right place to ignore this result is actually the CFStringGetBytes (which is returning a value passed through the closure here)